e2e/cli-e2e/vite.config.e2e.ts

```diff
@@ -1,19 +1,22 @@
 /// <reference types="vitest" />
-import { nxViteTsPaths } from '@nx/vite/plugins/nx-tsconfig-paths.plugin';
 import { defineConfig } from 'vite';
+import { tsconfigPathAliases } from '../../tools/vitest-tsconfig-path-aliases';
 
 export default defineConfig({
   cacheDir: '../../node_modules/.vite/cli-e2e',
-  plugins: [nxViteTsPaths()],
   test: {
+    reporters: ['default'],
     testTimeout: 120_000,
     globals: true,
+    alias: tsconfigPathAliases(),
+    pool: 'threads',
+    poolOptions: { threads: { singleThread: true } },
     cache: {
       dir: '../../node_modules/.vitest',
     },
     environment: 'node',
     include: ['tests/**/*.e2e.test.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
-    globalSetup: ['global-setup.e2e.ts'],
+    globalSetup: ['../../global-setup.e2e.ts'],
```

Comment:
The cwd of the nx executor changed from projectRoot to root i assume.

Reply:
Correct.